iam_openapi_spec.yml

```diff
@@ -1045,6 +1045,33 @@ paths:
         - basicAuth: [ ]
         - bearerAuth: [ ]
 
+  /authenticate:
+    post:
+      tags:
+        - User Authorization
+      summary: Authenticate a request
+      description: >
+        Authenticates the request and respond with token. Upon successful authentication,
+        - For basic auth as well as credential based bearer auth, this API generates a token and returns it.
+        - For JWT bearer auth, returns the same JWT token in response
```

Comment:
will this handle expiry check for JWT tokens ?

Reply:
Yes